src/lib/packet.h

```diff
@@ -435,6 +435,13 @@ typedef enum {
     PGP_SA_SM4 = 105 /* RNP extension - SM4 */
 } pgp_symm_alg_t;
 
+typedef enum {
+    PGP_BLOCK_CIPHER_MODE_NONE = 0,
+    PGP_BLOCK_CIPHER_MODE_CFB = 1,
+    PGP_BLOCK_CIPHER_MODE_CBC = 2,
+    PGP_BLOCK_CIPHER_MODE_OCB = 3,
+} pgp_block_cipher_t;
```

Comment:
I think the name `pgp_cipher_mode_t` would be slightly more accurate here. And then `PGP_CIPHER_MODE_CFB`, etc.